heka/files/toml/output/aggregator.toml

```diff
@@ -0,0 +1,7 @@
+{%- extends "heka/files/toml/output/tcp.toml" %}
+{%- block address -%}
+address = "{{ output.host }}:5565"
```

Comment:
At the end we will need to configure the firewall to allow aggregator traffic. It would be good to expose the port as param the same way there are exposed for other outputs.
you could at least add a FIXME and tackle this later